CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## 0.11.2 [#61](https://github.com/openfisca/openfisca-ops/pull/61)
```

Comment:
`0.11.1` seems to be missing.

However, I'm not sure what semantic versioning could mean in this context. No one is re-using `openfisca-ops` as a dependency. At most, re-users can get an idea of how to run OpenFisca by checking our conf, and can copy-paste some pieces of code.

Reply:
The reason for the changelog was to give visibility to changes done outside How could we log our changes to the server better ?
> proposition: 
```
# reason fof the change
- change made #1
- change made #2
....
```

Reply:
Ok, seems like a good idea 🙂